src/components/modals/ManagePDNTModal/ManagePDNTModal.tsx

```diff
@@ -319,6 +320,20 @@ function ManagePDNTModal() {
                                       pdntState!.records['@'].ttlSeconds,
                                     ],
                                   )
+                                : row.interactionType ===
+                                  PDNT_INTERACTION_TYPES.SET_TTL_SECONDS
+                                ? mapTransactionDataKeyToPayload(
+                                    CONTRACT_TYPES.PDNT,
+                                    row.interactionType,
+                                    [
+                                      '@',
+                                      pdntState!.records['@'].transactionId
+                                        .length
+                                        ? pdntState!.records['@'].transactionId
+                                        : STUB_ARWEAVE_TXID,
```

Comment:
i'm not a fan of stubbing the tx id

Reply:
niether am I but its required per contract code else we cannot se the ttl seconds 
![image](https://user-images.githubusercontent.com/85306700/235005831-99fdacfd-1f94-47ba-890e-5db51657a515.png)